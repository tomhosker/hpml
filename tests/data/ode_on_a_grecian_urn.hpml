Thou still unravished bride of quietness,
##TAB Thou foster-child of silence and slow time,
Sylvan historian, who canst thus express
##TAB A flowery tale more sweetly than our rhyme:
What leaf-fring'd legend haunts about thy shape
##TAB Of deities or mortals, or of both,
##TAB ##TAB In #PLACE{Tempe} or the dales of #PLACE{Arcady}?
##TAB What men or gods are these? What maidens loth?
What mad pursuit? What struggle to escape?
##TAB ##TAB What pipes and timbrels? What wild ecstasy?

Heard melodies are sweet, but those unheard
##TAB Are sweeter; therefore, ye soft pipes, play on;
Not to the sensual ear, but, more endeared,
##TAB Pipe to the spirit ditties of no tone:
Fair youth, beneath the trees, thou canst not leave
##TAB Thy song, nor ever can those trees be bare;
##TAB ##TAB Bold lover, never, never canst thou kiss,
Though winning near the goal yet, do not grieve;
##TAB She cannot fade, though thou hast not thy bliss,
##TAB ##TAB For ever wilt thou love, and she be fair!

Ah, happy, happy boughs! that cannot shed
##TAB Your leaves, nor ever bid the spring adieu;
And, happy melodist, unwearied,
##TAB For ever piping songs for ever new;
More happy love! more happy, happy love!
##TAB For ever warm and still to be enjoyed,
##TAB ##TAB For ever panting, and for ever young;
All breathing human passion far above,
##TAB That leaves a heart high-sorrowful and cloyed,
##TAB ##TAB A burning forehead, and a parching tongue.

Who are these coming to the sacrifice?
##TAB To what green altar, O mysterious priest,
Lead'st thou that heifer lowing at the skies,
##TAB And all her silken flanks with garlands drest?
What little town by river or sea shore,
##TAB Or mountain-built with peaceful citadel,
##TAB ##TAB Is emptied of this folk, this pious morn?
And, little town, thy streets for evermore
##TAB Will silent be; and not a soul to tell
##TAB ##TAB Why thou art desolate, can e'er return.

O attic shape! Fair attitude! with brede
##TAB Of marble men and maidens overwrought,
With forest branches and the trodden weed;
##TAB Thou, silent form, dost tease us out of thought
As doth eternity: cold pastoral!
##TAB When old age shall this generation waste,
##TAB ##TAB Thou shalt remain, in midst of other woe
Than ours, a friend to man, to whom thou say'st,
##TAB `Beauty is truth, truth beauty -- that is all
##TAB ##TAB Ye know on earth, and all ye need to know.'
