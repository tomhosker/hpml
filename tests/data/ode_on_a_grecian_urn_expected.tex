Thou still unravished bride of quietness,\\*
\vin Thou foster-child of silence and slow time,\\
Sylvan historian, who canst thus express\\
\vin A flowery tale more sweetly than our rhyme:\\
What leaf-fring'd legend haunts about thy shape\\
\vin Of deities or mortals, or of both,\\
\vin \vin In \textsc{Tempe} or the dales of \textsc{Arcady}?\\
\vin What men or gods are these? What maidens loth?\\
What mad pursuit? What struggle to escape?\\*
\vin \vin What pipes and timbrels? What wild ecstasy?\\!

Heard melodies are sweet, but those unheard\\*
\vin Are sweeter; therefore, ye soft pipes, play on;\\
Not to the sensual ear, but, more endeared,\\
\vin Pipe to the spirit ditties of no tone:\\
Fair youth, beneath the trees, thou canst not leave\\
\vin Thy song, nor ever can those trees be bare;\\
\vin \vin Bold lover, never, never canst thou kiss,\\
Though winning near the goal yet, do not grieve;\\
\vin She cannot fade, though thou hast not thy bliss,\\*
\vin \vin For ever wilt thou love, and she be fair!\\!

Ah, happy, happy boughs! that cannot shed\\*
\vin Your leaves, nor ever bid the spring adieu;\\
And, happy melodist, unwearied,\\
\vin For ever piping songs for ever new;\\
More happy love! more happy, happy love!\\
\vin For ever warm and still to be enjoyed,\\
\vin \vin For ever panting, and for ever young;\\
All breathing human passion far above,\\
\vin That leaves a heart high-sorrowful and cloyed,\\*
\vin \vin A burning forehead, and a parching tongue.\\!

Who are these coming to the sacrifice?\\*
\vin To what green altar, O mysterious priest,\\
Lead'st thou that heifer lowing at the skies,\\
\vin And all her silken flanks with garlands drest?\\
What little town by river or sea shore,\\
\vin Or mountain-built with peaceful citadel,\\
\vin \vin Is emptied of this folk, this pious morn?\\
And, little town, thy streets for evermore\\
\vin Will silent be; and not a soul to tell\\*
\vin \vin Why thou art desolate, can e'er return.\\!

O attic shape! Fair attitude! with brede\\*
\vin Of marble men and maidens overwrought,\\
With forest branches and the trodden weed;\\
\vin Thou, silent form, dost tease us out of thought\\
As doth eternity: cold pastoral!\\
\vin When old age shall this generation waste,\\
\vin \vin Thou shalt remain, in midst of other woe\\
Than ours, a friend to man, to whom thou say'st,\\
\vin `Beauty is truth, truth beauty -- that is all\\*
\vin \vin Ye know on earth, and all ye need to know.'